CHANGELOG.md

```diff
@@ -58,6 +59,18 @@
 - [v0.2.0](#v020)
 - [v0.1.0](#v010)
 
+## [v1.22.1]
+
+> Release date: 2023/06/22
```

Comment:
Can you please also add an entry for the diffs reference at the bottom of the file: https://github.com/Kong/deck/blob/b373907de6a271c364d1b2a357529f8dccf0f453/CHANGELOG.md?plain=1#L1257

Reply:
oh! sure thing

Reply:
done